tests/integration/pilot/validation_test.go

```diff
@@ -166,21 +166,22 @@ func TestEnsureNoMissingCRDs(t *testing.T) {
 			for _, r := range collections.Pilot.All() {
 				s := strings.Join([]string{r.Resource().Group(), r.Resource().Version(), r.Resource().Kind()}, "/")
 				recognized.Insert(s)
-			}
-			for _, gvk := range []string{
-				"networking.istio.io/v1beta1/Gateway",
-				"networking.istio.io/v1beta1/DestinationRule",
-				"networking.istio.io/v1beta1/VirtualService",
-				"networking.istio.io/v1beta1/WorkloadEntry",
-				"networking.istio.io/v1beta1/Sidecar",
-			} {
-				recognized.Insert(gvk)
+				for _, alias := range r.Resource().GroupVersionAliasKinds() {
+					s = strings.Join([]string{alias.Group, alias.Version, alias.Kind}, "/")
+					recognized.Insert(s)
+				}
 			}
 			// These CRDs are validated outside of Istio
 			for _, gvk := range []string{
+				"networking.istio.io/v1beta1/ServiceEntry",
+				"networking.istio.io/v1beta1/WorkloadGroup",
+				"networking.istio.io/v1beta1/EnvoyFilter",
```

Comment:
why are we removing these? they shouldn't be in `recognized` anyways since they don't exist...?

Reply:
oh we should add tests then. But Envoyfilter v1beta1  is definitely wrong, we should make sure the metadata.yaml is correct

Reply:
I see `v1beta1` alias for EnvoyFilter as well

Reply:
I nkow the alias is there, but the metadatal.yaml is wrong; EnvoyFilter v1beta1 doesn't exist.